assets/designer/environments/aml-designer/asset.yaml

```diff
@@ -0,0 +1,6 @@
+name: aml-designer
```

Comment:
please don't use aml, AzureML, microsoft, etc words in the asset name. you can find naming conventions in wiki

Reply:
Thanks, I just change it to designer-base